islandora.api.php

```diff
@@ -886,3 +886,41 @@ function hook_islandora_edit_datastream_registry_alter(&$edit_registry, $context
 function hook_islandora_repository_connection_construction_alter(RepositoryConnection $instance) {
   $instance->userAgent = "Tuque/cURL";
 }
+
+/**
+ * Allow a overridable backend for generating breadcrumbs.
+ *
+ * Stolen shamelessly from @adam-vessey.
+ *
+ * @return array
+ *   Should return an associative array mapping unique (module-prefixed,
+ *   preferably) keys to associative arrays containing:
+ *   - title: A human-readable title for the backend.
+ *   - callable: A PHP callable to call for this backend, implementing
+ *     callback_islandora_basic_collection_query_backends().
+ *   - file: An optional file to load before attempting to call the callable.
+ */
+function hook_islandora_breadcrumbs_backends() {
+  return array(
+    'awesome_backend' => array(
+      'title' => t('Awesome Backend'),
+      'callable' => callback_islandora_breadcrumbs_backends($object),
```

Comment:
Just taking a quick look, this may be kind of misleading... Or... At least a strange structure to offer as a example (a function call which returns a callable): Thinking this should maybe be:
```php
'callable' => 'callback_islandora_breadcrumbs_backends',
```